advocacy_docs/partner_docs/VisionXGuide/05-UsingVisionx.mdx

```diff
@@ -0,0 +1,432 @@
+---
+title: 'Using VisionX'
+description: 'Walkthroughs of multiple VisionX usage scenarios'
+---
+
```

Comment:
I think we need to be clearer with this.
Talk about migrating from Oracle to Postgres
Then using VisionX with EDB only.
Make is clear what we are going to show them and why they would use each section.  They don't need to do all of it.  Only what they need.